fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyzeStmt.java

```diff
@@ -63,48 +61,36 @@ public class AnalyzeStmt extends DdlStmt {
     // time to wait for collect  statistics
     public static final String CBO_STATISTICS_TASK_TIMEOUT_SEC = "cbo_statistics_task_timeout_sec";
 
-    public boolean isHistogram = false;
-
     private static final ImmutableSet<String> PROPERTIES_SET = new ImmutableSet.Builder<String>()
             .add(CBO_STATISTICS_TASK_TIMEOUT_SEC)
             .build();
 
     private static final Predicate<Long> DESIRED_TASK_TIMEOUT_SEC = (v) -> v > 0L;
 
-    public final boolean wholeTbl;
+    public boolean isWholeTbl;
```

Comment:
Is this variable really necessary?

Reply:
This is an existing parameter used by`MVAnalysisTask` to make logical judgment:  ”Analysis for the materialized  view, only gets constructed when the AnalyzeStmt is not set which columns to be analyzed“. It doesn't seem to need to be removed